admin_guide/upgrading_citus.rst

```diff
@@ -123,7 +123,7 @@ On the Coordinator Node
 
     CREATE TABLE public.pg_dist_partition AS SELECT * FROM pg_catalog.pg_dist_partition;
     CREATE TABLE public.pg_dist_shard AS SELECT * FROM pg_catalog.pg_dist_shard;
-    CREATE TABLE public.pg_dist_shard_placement AS SELECT * FROM pg_catalog.pg_dist_shard_placement;
+    CREATE TABLE public.pg_dist_placement AS SELECT * FROM pg_catalog.pg_dist_placement;
```

Comment:
Just dropping a note here that during the upgrade process this was the only notable issue I ran into. Basically I copy-pasted all the commands and just removed occurrences of the string `shard_` from them and things worked thereafter. This was as a result of upgrading from Citus 6.2/PostgreSQL 9.6 to Citus 7.0/PostgreSQL 9.6 and then finally Citus 7.0/PostgreSQL 10. I repeated the steps beginning with Citus 6.0 instead of 6.2 and the need still exists there, of course.